src/main/java/com/google/api/codegen/viewmodel/RetryCodesDefinitionView.java

```diff
@@ -18,13 +18,21 @@
 import com.google.common.collect.ImmutableSet;
 
 import io.grpc.Status.Code;
+import java.util.List;
+import javax.annotation.Nullable;
 
 @AutoValue
 public abstract class RetryCodesDefinitionView {
   public abstract String key();
 
+  @Nullable // Used in C#
+  public abstract String name();
+
   public abstract ImmutableSet<Code> codes();
 
+  @Nullable // Used in C#
+  public abstract List<String> codeNames();
```

Comment:
Could you move the retry param code from JavaGapicSurfaceTransformer into a separate transformer class that Java and C# could share?